### PR TITLE
Feature mes 1248 show journal slots basic new

### PR DESCRIPTION
**Pull request by rossfauldsbjss**
MES-1248 Amend journal page to show slots (raw json format). 
Data source is currently mocked with fixed JSON. 

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [x] Tested by QA
- [x] PO's approval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +74,7 @@
     "ionic-mocks-jest": "^1.3.3",
     "jest": "^23.6.0",
     "jest-preset-angular": "^6.0.1",
+    "json-schema-to-typescript": "^6.1.0",
```
Comment (ammarhaiderbjss) [inline package.json]: For awareness I think we may go down the path of centralising these JSON schemas and TS interfaces in another repo to be pulled in by the app/microservices. Can stay for now but we will probably remove later..

File: src/pages/journal/journal.ts [hunk 2]
```diff
@@ -1,11 +1,422 @@
...
 @IonicPage()
 @Component({
   selector: 'page-journal',
   templateUrl: 'journal.html'
 })
 export class JournalPage {
-  constructor(public navCtrl: NavController, public navParams: NavParams) {}
+  private journalJson: ExaminerWorkSchedule; // this should be populated by a call to a service
+  public journalJsonText: string; 
+  public journalSlot: any;
```
Comment (ammarhaiderbjss) [inline src/pages/journal/journal.ts]: Weird that the interface generator didn't create one for testSlot

**Comment by ammarhaiderbjss** [comment]
Conflicts

File: src/pages/journal/journal.ts (hunk 2 above)
Comment (johnmorleybjss) [reply src/pages/journal/journal.ts]: I think there may still be some refactoring to be done on the schema to ensure that we get as many types as possible.

**Pull request merged**